Leaflet/README.md

```diff
@@ -0,0 +1,7 @@
+# Leaflet API Integration App
+
+This project is intended to provide an easy-to-use example of how to integrate our Catalog & Image APIs into a Leaflet javascript application.
+
+To run the app:
+1. In file index.html, on line 95, add your Bird.i API token
```

Comment:
Maybe add a link to the specific line?

Reply:
Sure thing - I just need to merge it before I can get the link!